modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -468,7 +476,7 @@ definition {
 				key_tabName = "Account Name 1",
 				locator1 = "ObjectAdmin#ANY_TAB_ON_ENTRY");
 
-			AssertElementNotPresent(
+			AssertNotVisible(
```

Comment:
Why are we switching from AssertElementNotPresent to AssertNotVisible?

Reply:
hey @mateussandes23 can you test with AssertElementNotPresent and see if now it will work